### PR TITLE
Update metadata

### PR DESCRIPTION
**Pull request by davidwalter2**
- The meta data is updated in the HDF5Writer for [PR410](https://github.com/WMass/WRemnants/pull/410), since more information per channel is needed this information is now structured inside a dict in ["meta"]["channel_info"]

### CODE REVIEW

File: narf/combineutils.py [hunk 1]
```diff
@@ -126,13 +126,13 @@ def __init__(self, filename, pseudodata=None):
...
+                self.channel_info = self.metadata["channel_info"]
             else:
-                self.channel_axes = {
-                    "ch0": [hist.axis.Integer(0, self.nbins, underflow=False, overflow=False, name="obs")]
+                self.channel_info = {
+                    "ch0":{"axes": [hist.axis.Integer(0, self.nbins, underflow=False, overflow=False, name="obs")]}
                 }
                 if self.nbinsmasked > 0:
-                    self.channel_axes["ch1_masked"] = [hist.axis.Integer(0, self.nbinsmasked, underflow=False, overflow=False, name="masked")]
+                    self.channel_info["ch1_masked"] = {"axes": [hist.axis.Integer(0, self.nbinsmasked, underflow=False, overflow=False, name="masked")]}
```
Comment (bendavid) [inline narf/combineutils.py:135]: Would it make more sense for it to be called "ch0_masked"?

Comment (davidwalter2) [reply narf/combineutils.py:135]: Ah yes I think in the HDF5 writer we changed it to 0 but forget it here, I can can make another commit

Comment (davidwalter2) [reply narf/combineutils.py:135]: Hmm no in the HDF5 writer we also name it "ch1_masked" because "ch0" is already the one that is not masked.

**Comment by bendavid** [comment]
Ok, I was thinking of this in terms of "ch0_masked" is the masked companion to "ch0"

**Pull request merged**